tool/site/client-react/src/pages/homePage.tsx

```diff
@@ -0,0 +1,28 @@
+import {ActionLink, Label} from "nhsuk-react-components"
+import * as React from "react"
+import {useContext} from "react"
+import {AppContext} from "../index"
+
+const HomePage: React.FC = () => {
+  const {baseUrl} = useContext(AppContext)
+
+  return (
+    <>
```

Comment:
do we still want the 'change-auth' / 'login' button at the top?

Reply:
good question, i think whilst we're changing our login flow for attended/unattended we hold off making a decision on how we want to handle login flows/ui till this is stable